protocols/v2/subprotocols/common-messages/src/setup_connection.rs

```diff
@@ -16,38 +16,50 @@ use core::convert::TryInto;
 #[cfg(feature = "with_serde")]
 use serde_repr::*;
 
-/// ## SetupConnection (Client -> Server)
-/// Initiates the connection. This MUST be the first message sent by the client on the newly
-/// opened connection. Server MUST respond with either a [`SetupConnectionSuccess`] or
-/// [`SetupConnectionError`] message. Clients that are not configured to provide telemetry data to
-/// the upstream node SHOULD set device_id to 0-length strings. However, they MUST always set
-/// vendor to a string describing the manufacturer/developer and firmware version and SHOULD
-/// always set hardware_version to a string describing, at least, the particular hardware/software
-/// package in use.
+/// Used by a downstream role to initiate a Stratum V2 channel connection with an upstream role.
 ///
+/// This is usually the first message sent by a downstream role on a newly opened connection,
+/// after completing the handshake process.
+///
+/// Downstream that do not wish to provide telemetry data to the upstream role **should** set
```

Comment:
```suggestion
/// Downstreams that do not wish to provide telemetry data to the upstream role SHOULD set
```

Reply:
The "**word**"  makes the word **bold** on docs.rs, which I think makes more sense than all cap..